File: reverseArray/reverseArray.py
array = [int(x) for x in input("Please enter Array: ").split()]

## Hard Way
i = len(array) - 1
j = 0
while(j < i):
   temp = array[i]
   array[i] = array[j]
   array[j] = temp
   i -= 1
   j += 1
print ('Hard Way: ',array)

## Python way
array.reverse()
print ('Python way: ',array)

File: reverseArray/reverseArray.cpp
#include <iostream>

int main()
{
   int a[10];
   std::cout << "Enter input array: " << std::endl;
   for(int i = 0; i < 10; i++)
      std::cin >> a[i];  
   int len = sizeof(a)/sizeof(*a);
   int i = len - 1;
   int j = 0;

   while(j < i)
   {
      int temp = a[i];
      a[i] = a[j];
      a[j] = temp;
      i--;
      j++;
   }

   std::cout << "Reversed Array: ";
   for(int i = 0; i < 10;i++)
      std::cout << a[i] <<  " ";
   std::cout << std::endl;

   return 0;
}
